src/components/states/instructions.js

```diff
@@ -6,22 +6,44 @@ AFRAME.registerComponent('instructions', {
     this.startButtonEl = document.querySelector('.start-button');
     this.startButtonEl.querySelector('span').innerHTML = 'Loading';
     this.startButtonEl.querySelector('span').classList.add('loading');
-    this.onClicked = this.onClicked.bind(this);
+    this.startReplaying = this.startReplaying.bind(this);
     this.loading = true;
     this.el.sceneEl.addEventListener('loaded', this.downloadDance.bind(this));
   },
 
+  handleDanceData: function (data) {
+    this.danceData = data;
+
+    var sceneEl = this.el.sceneEl;
+    var inVR = sceneEl.is('vr-mode');
+
+    if (!sceneEl.hasLoaded) {
+      sceneEl.addEventListener('loaded', this.handleDanceData.bind(this));
+      return;
+    }
+
+    if (inVR) {
+      this.startReplaying();
+    } else {
```

Comment:
I would attach a listener to `enter-vr` on the init method that starts replaying if any data has loaded.